watertap/core/membrane_channel0d.py

```diff
@@ -64,6 +65,32 @@ def add_geometry(
         Returns:
             None
         """
+        # volume will be added be calling add_geometry from ControlVolume0D
+        if not hasattr(self, "length") and not hasattr(self, "width"):
+            self._add_var_reference(length_var, "length", "length_var")
+            self._add_var_reference(width_var, "width", "width_var")
+
+        if hasattr(self, "length") and hasattr(self, "width"):
+            super().add_geometry()
+            units_meta = self.config.property_package.get_metadata().get_derived_units
+
+            if not hasattr(self, "channel_height"):
+                self.channel_height = Var(
+                    initialize=1e-3,
+                    bounds=(1e-4, 5e-3),
+                    domain=NonNegativeReals,
+                    units=units_meta("length"),
+                    doc="membrane-channel height",
+                )
+
+            # # TODO: negating spacer volume for now as it can be assumed negligible (Park et al., 2020: https://doi.org/10.1016/j.desal.2020.114625). Revisit.
+            # # Note: considering that this volume relationship holds for both flat_sheet and spiral_wound types
+            @self.Constraint(
+                self.flowsheet().config.time, doc="Membrane-channel volume"
+            )
+            def eq_volume(b, t):
+                return b.volume[t] == b.length * b.width * b.channel_height
```

Comment:
```suggestion
                return b.volume[t] == (b.length * b.width * b.channel_height)*spacer_porosity
```
But since spacer porosity isn't a var on this level, maybe we take care of this in the RO unit model?

Reply:
This is partly why I am pushing this for a later PR. I want to first get RO working as anticipated with dynamics and has_holdup set to True, making any _necessary changes_ to model structure as we go. 

Then, once that is set, we can come back to this and incorporate spacer volume. Also, when we _do get back to this_, spacer_porosity seems like an approximation. Do you think we should settle for that and assume negligible difference in comparison with true spacer volume?